frontend-react/src/hooks/UseFilterManager.ts

```diff
@@ -0,0 +1,131 @@
+import React, { useMemo, useState } from "react";
+
+export type SortOrder = "ASC" | "DESC";
+export type PageSize = 10 | 25 | 50 | 100;
```

Comment:
Interesting that you picked a subset of possible page sizes. Is this just a way to make sure it's a reasonable value (not too low/not too high)?